src/client/containers/CareerPage/CareerPage.component.js

```diff
@@ -70,8 +75,19 @@ const CareerPage = ({ jobTitle }) => {
             </div>
           </div>
           <div className="buttons-result-page">
-            <Button label="Retake quiz" size="big" color="grey" />
-            <Button label="Elevator pitch" size="big" />
+            <Link to="/quiz">
+              <Button
+                className="first-button"
+                label="Retake quiz"
+                size="big"
+                color="grey"
+                isMono={true}
+                onClick={handleClick}
+              />
+            </Link>
+            <Link to="/elevatorpitch">
+              <Button label="Elevator pitch" size="big" isMono={true} />
+            </Link>
```

Comment:
This is not the correct way to write this. think of it as it ends up being HTML and you should not wrap an <a> tag around a <button> tag.

https://stackoverflow.com/questions/6393827/can-i-nest-a-button-element-inside-an-a-using-html5

If you want to navigate to a page using a button this is the correct way to do so :).

https://reactrouter.com/web/api/Hooks/usehistory